client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -111,29 +123,90 @@ export default defineComponent({
       return 'mdi-image-multiple';
     }
 
+    async function preloadCheck(recent: JsonMetaCache) {
+      //Attempts to preload the data to see if there are any isues
+      try {
+        await api.checkDataset(recent.id);
```

Comment:
The most common error I can think of involves the case where the user changed the location of the source media, which need to be re-located to launch the viewer.  This doesn't check that, right?

Reply:
Yeah I didn't know how far to go for that.  It checks to make sure the `originalBasePath` (folder which has the source video/images) exists.  So it goes part of the way to make sure the path exists.  I didn't want to go through and check all of the individual images.

Reply:
I think this is reasonable for this PR.  I'll mark this one resolved.